libraries/AP_RangeFinder/AP_RangeFinder_BBB_PRU.cpp

```diff
@@ -51,14 +51,25 @@ AP_RangeFinder_BBB_PRU::AP_RangeFinder_BBB_PRU(RangeFinder::RangeFinder_State &_
 bool AP_RangeFinder_BBB_PRU::detect()
 {
     bool result = true;
-    uint32_t mem_fd;
-    uint32_t *ctrl;
-    void *ram;
-
-    mem_fd = open("/dev/mem", O_RDWR | O_SYNC | O_CLOEXEC);
-    ctrl = (uint32_t*)mmap(0, 0x1000, PROT_READ | PROT_WRITE, MAP_SHARED, mem_fd, PRU0_CTRL_BASE);
-    ram = mmap(0, PRU0_IRAM_SIZE, PROT_READ | PROT_WRITE, MAP_SHARED, mem_fd, PRU0_IRAM_BASE);
 
+    uint32_t mem_fd = open("/dev/mem", O_RDWR | O_SYNC | O_CLOEXEC);
+    if(mem_fd == -1)
```

Comment:
style-guide requires that the brace goes on the same line as the `if`, and a space between the if and the pren.